data/strings/strings.txt

```diff
@@ -19337,7 +19339,7 @@
 
   [crash_reports_description]
     tags = android
-    en = We may use your data to improve Organic Maps experience. Changes will take effect after you restart the app.
+    en = We may use your data to improve Organic Maps. Changes will take effect after you restart the app.
```

Comment:
```suggestion
    en = We may use your crash reports to improve Organic Maps. Changes will take effect after you restart the app.
```

Reply:
Agreed on @biodranik  wording suggestion.

I do not understand what changes take effect.  opting into or out of crash reports  I would expect to be immediate.  I would expect the time from submitting a crash report to a resulting fix being available to me much longer than the time it takes to restart :-)  But seriously, I have no idea what this means.